src/app/models/form_issue.server.model.ts

```diff
@@ -36,7 +40,22 @@ const FormIssueSchema = new Schema<IFormIssueSchema, IFormIssueModel>(
 FormIssueSchema.index({ formId: 1 })
 
 /**
- * Form Issue Schema
+ * Returns a cursor for all issues for the form with formId.
+ * @param formId the form id to return the cursor for
+ * @returns a cursor to the issue retrieved
+ */
+FormIssueSchema.statics.getIssueCursorByFormId = function (
+  formId: string,
+): QueryCursor<FormIssueStreamData> {
+  return this.find({ formId }, { issue: 1, email: 1, created: 1 })
```

Comment:
Is the selection portion required to be specified here? Wondering if we need to whitelist the fields here

Reply:
It's not really nice and not needed to throw the entire database columns to FE, exposing the internal metadata. So I just project whatever is needed.

Reply:
Should this be done at the controller or service instead?

Reply:
good point, let me change this.